src/server/routes/unitReadings.js

```diff
@@ -161,8 +179,9 @@ function createRouter() {
 		} else {
 			const conn = getConnection();
```

Comment:
I think `conn` is no longer necessary here. Do you think we should remove it? The same for lines 193, 206, and 219.

Reply:
I agree and it was removed.